main.cpp

```diff
@@ -2,15 +2,57 @@
 #include "rc.h"
 #include <filesystem>
 
+Gdiplus::Image* Image = nullptr;
+TCHAR imagepath[MAX_PATH] = { 0 };
+
+void file_open(HWND parent)
+{
+	TCHAR path[MAX_PATH]; *path = 0;
+	ZeroMemory(path, sizeof(path));
+	TCHAR filter[] = _T("Image\0*.png;*\0*.jpeg;*\0*.bmp;*\0*.gif;*\0*.tiff;*\0*.emf\0");
+	OPENFILENAME ofn{ sizeof OPENFILENAME };
+	ofn.hwndOwner = parent;
+	ofn.lpstrFile = path;
+	ofn.nMaxFile = MAX_PATH;
+	ofn.lpstrFilter = filter;
+	ofn.Flags = OFN_HIDEREADONLY;
+	if (::GetOpenFileName(&ofn)) {
+		Image = Gdiplus::Image::FromFile(path);
```

Comment:
A što ako je `Image` već učitan? Niste obrisali prethodni.
A nikada ni ne brišete ovaj `Image` (prijedlog: koristite `std::unique_ptr`).